src/hostname/hostnamectl.c

```diff
@@ -299,6 +308,7 @@ static int show_all_names(sd_bus *bus) {
                 { "HomeURL",                   "s", NULL, offsetof(StatusInfo, home_url)        },
                 { "HardwareVendor",            "s", NULL, offsetof(StatusInfo, hardware_vendor) },
                 { "HardwareModel",             "s", NULL, offsetof(StatusInfo, hardware_model)  },
+                { "FirmwareVersion",           "s", NULL, offsetof(StatusInfo, firmware_version) },
```

Comment:
please re-break the whole table (ie, re-align the closing brackets in the lines above)

Reply:
done

Reply:
Not aligned yet...